src/recon_buildblock/ForwardProjectorByBin.cxx

```diff
@@ -156,7 +156,7 @@ forward_project(RelatedViewgrams<float>& viewgrams,
      iter != viewgrams.end();
      ++iter)
       {
-    ViewSegmentNumbers vs(iter->get_view_num(), iter->get_segment_num());
+    ViewSegmentTOFNumbers vs(iter->get_view_num(), iter->get_segment_num());
```

Comment:
Add `timing_pos_num`?

Reply:
This sits within `#ifdef STIR_PROJECTORS_AS_V3`. No clear need to modify this yet.

Reply:
ok. let's call `error`